src/Properties.ts

```diff
@@ -0,0 +1,28 @@
+import { Logger } from '@hmcts/nodejs-logging'
+import * as fs from 'fs'
+
+const log = Logger.getLogger('applicationRunner')
+
+export function addTo (config: any, mountPoint: fs.PathLike = '/mnt/secrets/', propertiesPrefix: string = 'keyVault') {
```

Comment:
ideally this would be `secrets` but this repo is called properties...
```suggestion
export function addTo (config: any, mountPoint: fs.PathLike = '/mnt/secrets/', propertiesPrefix: string = 'secrets') {
```

either way i don't think keyVault is the correct previx

Reply:
sure ... althougth this will also allow values secrets and certificates.

Reply:
maybe should just change this to take the last folder in the path ?

Reply:
changed this to reflect the mount point ... i.e. /mnt/secret will have secret.xxx and /mnt/certs will have certs.xxx